codebuild/release/release-prod.yml

```diff
@@ -2,13 +2,15 @@ version: 0.2
 
 env:
   secrets-manager:
+    GPG_KEY: Maven-GPG-Keys-Credentials:Keyname
+    GPG_PASS: Maven-GPG-Keys-Credentials:Passphrase
     SONA_USERNAME: Sonatype-Team-Account:Username 
-    SONA_PASS: Sonatype-Team-Account:Password
+    SONA_PASSWORD: Sonatype-Team-Account:Password
 
 phases:
   install:
     runtime-versions:
-      java: openjdk8
+      java: openjdk11
```

Comment:
For posterity, per offline discussion: since the validation in these release steps is testing against all supported versions of Java (8 and 11, corretto and openjdk), bumping to 11 for the actual building of artifacts is a reasonable update.